invenio_rdm_records/fixtures/communities.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Communities fixture module."""
+
+from pathlib import Path
+
+from invenio_communities.fixtures.tasks import create_demo_community
+
+from .fixture import FixtureMixin
+
+
+class CommunitiesFixture(FixtureMixin):
+    """Communities fixture."""
+
+    def __init__(self, logo_path, search_paths, filename):
+        """Initialize the communities fixture."""
+        super().__init__(search_paths, filename)
+        self.logo_path = logo_path
+
+    def create(self, entry):
+        """Load a single community."""
+        logo = entry.pop("logo", None)
+
+        if logo:
+            logo_file_path = self.logo_path / logo
+            with open(Path(logo_file_path), "rb") as filestream:
+                create_demo_community(entry, filestream)
```

Comment:
major: celery tasks can be called in 2 different ways:

a) as a function, like in this case and in line 34. That means they will run "eagerly" or "on the spot".
b) delayed or "asynchronously", for that we have to call them as `create_demo_community.delay(....)`. That is for example why the `Fixtures` class has a `delay` parameter. See more [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/fixtures/vocabularies.py#L346).

Extra information linking this and my previous comment. What would happen if you open the descriptor here (as done in line 31, but then call the task with `.delay`? Would it work?